website/docs/components/code-block/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,15 @@
+## Conformance rating
+
+<Doc::Badge @type="success">Conformant</Doc::Badge>
+
+When used as recommended, there should not be any WCAG conformance issues with this component.
```

Comment:
Isn't the button border contrast a known a11y issue? Should it be documented here?

Reply:
If we are to document that here, we should also likely do it in the Button component.

Reply:
The icon itself has accessible contrast and I remember Jory tweaked the border contrast from the beginning and @MelSumner seemed satisfied? (@MelSumner please add a new comment if you think we should document anything in the accessibility tab around this though.)